tests/test_utils.py

```diff
@@ -160,6 +160,83 @@ def test_request_user_has_no_implicit_access_via_jwt_no_context(self):
             'some_context'
         )
 
+    def test_user_with_multiple_contexts_has_access_via_jwt(self):
+        """
+        Helper function should discern what roles user has based on role data
+        in jwt, and then return true if any of those match the role we're
+        asking about. This case handles checking if each available context
+        matches wben the user has access to multiple contexts.
+        """
+        toy_decoded_jwt = {
+            "roles": [
+                "coupon-manager:some_context",
+                "coupon-manager:some_other_context"
+            ]
+        }
+        assert request_user_has_implicit_access_via_jwt(
+            toy_decoded_jwt,
+            'coupon-management',
+            'some_context'
+        )
+
+        assert request_user_has_implicit_access_via_jwt(
+            toy_decoded_jwt,
+            'coupon-management',
+            'some_other_context'
+        )
+
+    def test_user_multiple_contexts_has_access_via_jwt_with_all_access_context(self):
```

Comment:
is this test a copy paste error? looks to be testing the same thing as the previous test (title and docstring looks to be describing the next test)

Reply:
you were right - should be fixed now